winml/adapter/winml_adapter_apis.h

```diff
@@ -73,9 +73,9 @@ ORT_API_STATUS(SessionRegisterCustomRegistry, _In_ OrtSession* session, _In_ IML
 
 // Dml methods (TODO need to figure out how these need to move to session somehow...)
 // ORT_API_STATUS(DmlExecutionProviderSetDefaultRoundingMode, _In_ const OrtExecutionProvider* dml_provider, _In_ bool is_enabled);
-// ORT_API_STATUS(DmlExecutionProviderFlushContext, _In_ OrtSession* session, _In_ OrtExecutionProvider * dml_provider);
-// ORT_API_STATUS(DmlExecutionProviderTrimUploadHeap, _In_ const OrtExecutionProvider* dml_provider);
-// ORT_API_STATUS(DmlExecutionProviderReleaseCompletedReferences, _In_ const OrtExecutionProvider* dml_provider);
+ORT_API_STATUS(DmlExecutionProviderFlushContext, _In_ OrtExecutionProvider * dml_provider);
```

Comment:
>DmlExecutionProviderFlushContext [](start = 15, length = 32)

can you call these apis in onnxruntimeengine.cpp?

Reply:
yep!